.github/workflows/release-kotlin-desktop.yml

```diff
@@ -74,8 +74,14 @@ jobs:
     steps:
       - uses: RDXWorks-actions/checkout@main
 
-      - uses: RDXWorks-actions/download-artifact@main
+      - uses: RDXWorks-actions/download-artifact-v4@main
         with:
+          name: aarch64-apple-darwin
+          path: artifacts
+
+      - uses: RDXWorks-actions/download-artifact-v4@main
+        with:
+          name: x86_64-unknown-linux-gnu
```

Comment:
should this really be `x86_64-unknown-linux-gnu` but the one above is `aarch64-apple-darwin` ? looks wrong with the assymetry?